hoomd/mpcd/ATCollisionMethod.cc

```diff
@@ -17,8 +17,9 @@ mpcd::ATCollisionMethod::ATCollisionMethod(std::shared_ptr<SystemDefinition> sys
                                            uint64_t period,
                                            int phase,
                                            std::shared_ptr<Variant> T)
-    : mpcd::CollisionMethod(sysdef, cur_timestep, period, phase), m_T(T)
+    : mpcd::CollisionMethod(sysdef, cur_timestep, period, phase)
     {
+    setTemperature(T);
```

Comment:
With T in the base constructor:
```suggestion
    : mpcd::CollisionMethod(sysdef, cur_timestep, period, phase, T)
    {
```